pkg/rest/config_upload.go

```diff
@@ -155,9 +156,9 @@ func upsertDynatraceObject(client *http.Client, fullUrl string, objectName strin
 
 func joinUrl(urlBase string, path string) string {
 	if strings.HasSuffix(urlBase, "/") {
-		return urlBase + path
```

Comment:
do we have any existing tests for this/could we add tests covering the special char case, so we notice if we break it again?